File: task-03/Go/2/Sub-Task2.go
package main

import (
	"fmt"
	"os"
)

func main() {
	inputContent, err := os.ReadFile("input.txt")
	if err != nil {
		fmt.Println("Error reading input file:", err)
		return
	}

	err = os.WriteFile("output.txt", inputContent, 0644)
	if err != nil {
		fmt.Println("Error writing to output file:", err)
		return
	}

	fmt.Println("Content successfully copied to output.txt")
}

File: task-03/Javascript/2/Sub-Task2.js
const fs = require('fs');

function readAndWriteFile(inputFile, outputFile) {
    fs.readFile(inputFile, 'utf8', (err, data) => {
        if (err) {
            console.error(`Error reading file ${inputFile}:`, err);
            return;
        }
        fs.writeFile(outputFile, data, (err) => {
            if (err) {
                console.error(`Error writing file ${outputFile}:`, err);
                return;
            }
            console.log(`Content successfully copied from ${inputFile} to ${outputFile}.`);
        });
    });
}

readAndWriteFile('input.txt', 'output.txt');